CMakeLists.txt

```diff
@@ -22,7 +22,35 @@ option(SOCI_SHARED "Enable build of shared libraries" ON)
 option(SOCI_STATIC "Enable build of static libraries" ON)
 option(SOCI_TESTS "Enable build of collection of SOCI tests" ON)
 option(SOCI_ASAN "Enable address sanitizer on GCC v4.8+/Clang v 3.1+" OFF)
-
+option(SOCI_LTO "Enable link time optimization" ON)
```

Comment:
I'm not sure about the default value here. Wouldn't it apply to debug builds too, where we definitely don't want to use it by default?

Reply:
I didn't take that into consideration.

I would set it to off by default.

Reply:
Done